packages/editor-ui/src/stores/nodeCreator.ts

```diff
@@ -339,8 +339,7 @@ export const useNodeCreatorStore = defineStore(STORES.NODE_CREATOR, {
 				if (!nodeType) return [];
 
 				const { workflowTriggerNodes } = useWorkflowsStore();
-				const isTrigger =
-					nodeType.toLocaleLowerCase().includes('trigger') || nodeType === WEBHOOK_NODE_TYPE;
+				const isTrigger = useNodeTypesStore().isTriggerNode(nodeType);
```

Comment:
I'd save `useNodeTypesStore()` call result to a variable but for that you should refactor the creation of this store to use a callback function that returns the object (which is now passed directly as the second argument of `defineStore`)

Reply:
Never mind, maybe switching to the `setup` store creation pattern is not what you want to do just right now :)

Reply:
Thanks! Exactly